tools/release.sh

```diff
@@ -71,80 +73,82 @@ elif [ $keycount -ne 1 ]; then
   done
 fi
 
-gpgfing=$(gpg --keyid-format 0xLONG --fingerprint $gpgkey | grep 'Key fingerprint =' | awk -F' = ' '{print $2}' | tr -d ' ')
+gpgfing=$(gpg --keyid-format 0xLONG --fingerprint "$gpgkey" | grep 'Key fingerprint =' | awk -F' = ' '{print $2}' | tr -d ' ')
 
-if ! test "$(grep $gpgfing README.md)"; then
+if ! test "$(grep "$gpgfing" README.md)"; then
   echo 'Error: this GPG key fingerprint is not listed in ./README.md'
   exit 1
 fi
 
 echo "Using GPG key: $gpgkey"
 echo "  Fingerprint: $gpgfing"
 
-function checktag {
-  local version=$1
+checktag() {
+  # local version=$1
 
-  if ! git tag -v $version 2>&1 | grep "${gpgkey}" | grep key > /dev/null; then
-    echo "Could not find signed tag for \"${version}\" or GPG key is not yours"
+  if ! git tag -v "$1" 2>&1 | grep "${gpgkey}" | grep key > /dev/null; then
+    echo "Could not find signed tag for \"${1}\" or GPG key is not yours"
     exit 1
   fi
 }
 
 ################################################################################
 ## Create and sign checksums file for a given version
 
-function sign {
-  echo -e "\n# Creating SHASUMS256.txt ..."
+sign() {
+  printf "\n# Creating SHASUMS256.txt ..."
 
-  local version=$1
+  # local version=$1
 
-  ghtaggedversion=$(curl -sL https://raw.githubusercontent.com/nodejs/node/${version}/src/node_version.h \
+  ghtaggedversion=$(curl -sL https://raw.githubusercontent.com/nodejs/node/"${1}"/src/node_version.h \
       | awk '/define NODE_(MAJOR|MINOR|PATCH)_VERSION/{ v = v "." $3 } END{ v = "v" substr(v, 2); print v }')
-  if [ "${version}" != "${ghtaggedversion}" ]; then
+  if [ "${1}" != "${ghtaggedversion}" ]; then
```

Comment:
Why the expansion + quoting and not the short form? `"$1"` should work just as well as `"${1}"`.

Reply:
No reason, I did a lazy search and replace, and didn't bother to check if it actually make sense. Let me change that.